Add info on applying reviews in the contribution guide (#6783)
ppy/osu-wiki: wiki/osu!_wiki/Contribution_guide/Best_practices/en.md

  - [Automatic resolution of relevant issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- Make sure to tick the `Allow edits from maintainers` checkbox, as it will allow the wiki maintainers to help you improve the pull request when necessary

## Applying reviews

> Reviews are best applied directly through the github web interface. Use the `Add suggestion to batch` button when in the `Files changed` tab to apply multiple reviews simultaneusly.

Author: ```suggestion
Reviews are best applied directly through the GitHub web interface. Use the `Add suggestion to batch` button when in the `Files changed` tab to apply multiple reviews simultaneously.
```

> ![][img/apply-batch-review.gif]

Author: alternative text for when the image doesn't load

```suggestion
![Applying a batch review][img/apply-batch-review.gif]
```

> You may also use the `Commit suggestion` button to apply a single suggestion individually, but don't forget to set a descriptive commit message. Keep in mind however, that fewer related commits are better than lots of small ones.

Author: i don't quite think that made sense...

```suggestion
You may also use the `Commit suggestion` button to apply a single suggestion individually, but don't forget to set a descriptive commit message. Keep in mind however, that fewer commits, each with related changes, are better than lots of small ones.
```

> You may also use the `Commit suggestion` button to apply a single suggestion individually, but don't forget to set a descriptive commit message. Keep in mind however, that fewer commits are better than lots of small ones.

Reviewer: or something like that?
```suggestion
You may also use the `Commit suggestion` button to apply a single suggestion individually, provided that you make commits sparingly and [with informative messages](#making-edits).
```

![][img/apply-single-review.gif]

> Using this system will automatically mark suggestions as resolved. When applying reviews manually (e.g. when the reviewer didnt add a direct suggestion), mark them as resolved *after committing the change* to prevent forgetting any. Applying reviews automatically is preferred, as it prevents errors and alleviates reviewers from potentially needing to check if their review was applied correctly, especially when there are many changes.

Author: ```suggestion
Using this system will automatically mark suggestions as resolved. When applying reviews manually (e.g. when the reviewer didn't add a direct suggestion), mark them as resolved *after committing the change* to prevent forgetting any. Applying reviews automatically is preferred, as it prevents errors and alleviates reviewers from potentially needing to check if their review was applied correctly, especially when there are many changes.
```

> Using this system will automatically mark suggestions as resolved. When applying reviews manually (e.g. when the reviewer didn't add a direct suggestion), mark them as resolved *after committing the change* to prevent forgetting any. Applying reviews automatically is preferred, as it prevents errors and alleviates reviewers from potentially needing to check if their review was applied correctly, especially when there are many changes.

Reviewer: the last sentence is overloaded, mind simplifying it?

Author: ```suggestion
Using this system will automatically mark suggestions as resolved. When applying reviews manually (e.g. when the reviewer didn't add a direct suggestion), mark them as resolved *after committing the change* to prevent forgetting any. Letting GitHub apply reviews automatically is preferred, as it ensures that suggestions are applied correctly and prevents any manual copy errors.
```

Reviewer: yep
